packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/pubspec.yaml

```diff
@@ -6,8 +6,8 @@ environment:
   sdk: ">=2.16.0 <3.0.0"
 
 dependencies:
-  cloud_firestore: ^4.3.1
-  cloud_firestore_odm: any
+  cloud_firestore: ^4.3.2
```

Comment:
The `cloud_firestore` pubspec hasn't updated 🤔

Reply:
I can see it has?

Reply:
ah, my bad 😓